brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/Phrase.kt

```diff
@@ -1,15 +1,37 @@
 package no.nav.pensjon.brev.template
 
-import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
-import no.nav.pensjon.brev.template.dsl.ParagraphOnlyScope
-import no.nav.pensjon.brev.template.dsl.TextOnlyScope
+import no.nav.pensjon.brev.template.dsl.*
 
 abstract class TextOnlyPhrase<Lang : LanguageSupport> {
     abstract fun TextOnlyScope<Lang, Unit>.template()
     fun apply(scope: TextOnlyScope<in Lang, *>) {
         TextOnlyScope<Lang, Unit>().apply { template() }.elements
             .forEach { scope.addTextContent(it) }
     }
+
+    fun applyToParagraphScope(scope: ParagraphScope<in Lang, *>) {
+        TextOnlyScope<Lang, Unit>().apply { template() }.elements
```

Comment:
Det hadde vært så mye bedre om vi fikk til samme arve-hierarki for fraser som vi har i scopes. Men jeg mener å huske det var en grunn til at vi ga opp på det.

Reply:
Ser ikke helt grunnen til at vi har textOnly og plainTextOnly egentlig. Husker vi ikke skulle lekke text-funksjoner helt ut til outline scope f.eks, men det bør vel gå fint om paragraphOnlyScope arver fra textScope som arver fra plainTextScope, også sletter vi textOnly og plainTextOnly scopes. Skal gjøre noen forsøk og se om jeg finner ut av hvorfor vi hadde de, eller om vi bare har kopiert metodikken nedover og ikke tenkt så mye over det.